main/reference/zoe-api/zoe-contract-facet.md

```diff
@@ -102,6 +102,21 @@ The optional **customDetails** argument is included in the **Invitation**'s
 The optional **proposalShape** argument can be used to describe the required and allowed components of each proposal.
 Proposals that don't match the pattern will be rejected by Zoe without even being sent to the contract.
 
+Patterns are constructed using the
+**[M](https://endojs.github.io/endo/interfaces/_endo_patterns.PatternMatchers.html)** (for '**M**atcher') object.
```

Comment:
My impression is that these are not yet intended for wide dissemination. Building the docs was one step but publishing was another.

Oh... but now I find:

 - https://github.com/endojs/endo/pull/1883
 
 So we're committing to the `endojs.github.io` domain externally? Seems worth getting product / IT buy-in.
 @turadg what's your sense of where we are with this?

Reply:
I agree it's worth getting cross-functional buy-in before linking to it from https://docs.agoric.com/

Reply:
@ivanlei , @sufyaankhan support this, for now; getting better integration with the docs site remains a goal.